www-apps/prowlarr/files/prowlarr.init

```diff
@@ -0,0 +1,7 @@
+#!/sbin/openrc-run
+
+pidfile="/run/prowlarr.pid"
+command="/opt/prowlarr/Prowlarr"
+command_user="${RC_SVCNAME}:${RC_SVCNAME}"
```

Comment:
prowlarr.init and radarr.init have `command_user="${RC_SVCNAME}:${RC_SVCNAME}"` , but sonarr has `command_user="sonarr:sonarr"`. It's just inconsistent.

Reply:
> It's just inconsistent.

I also despise inconsistency. I've used the name (not the variable) in all now.